index.bs

```diff
@@ -2900,9 +2902,17 @@ of running the following steps:
 
 <h3 id="obtain-aggregatable-report-aggregation-service-payloads">Obtaining an aggregatable report's aggregation service payloads</h3>
 
-To <dfn>obtain the public key for encryption</dfn> given an [=aggregation coordinator=]
-<var ignore=''>aggregationCoordinator</var>, asynchronously return a user-agent-determined public key
-or an error in the event that the user agent failed to obtain the public key.
+To <dfn>obtain the public key for encryption</dfn> given an [=aggregation coordinator=] |aggregationCoordinator|:
+
+1. [=Assert=]: |aggregationCoordinator| is not an [=opaque origin=].
+1. Let |url| be a new [=URL record=].
+1. Set |url|'s [=url/scheme=] to |aggregationCoordinator|'s [=origin/scheme=].
+1. Set |url|'s [=url/host=] to |aggregationCoordinator|'s [=origin/host=].
+1. Set |url|'s [=url/port=] to |aggregationCoordinator|'s [=origin/port=].
+1. Let |path| be «"`.well-known`", "`aggregation-service`", "`public-keys`"».
```

Comment:
Is the `|path|` variable necessary? It seems like we can just do `Set |url|'s [=url/path=] to...`